src/index.js

```diff
@@ -30,33 +31,36 @@ ReactDOM.render(
 
 function App() {
   return (
-    <div>
-      {/* Add the Route in to App */}
-      <header>
-        <NavBar />
-      </header>
+    <>
+      <div>
+        {/* Add the Route in to App */}
+        <header>
+          <NavBar />
+        </header>
 
-      <Switch>
-        <Route path="/register">
-          <RegisterForm />
-        </Route>
-        <Route path="/login">
-          <LoginForm />
-        </Route>
-        <Route path="/dashboard">
-          <Dashboard />
-        </Route>
-        <Route path="/dashboard-chart">
-          <DashboardChart />
-        </Route>
-        <Route path="/about">
-          <About />
-        </Route>
-        <Route path="/">
-          <Container />
-        </Route>
-      </Switch>
-      <Footer />
-    </div>
+        <Switch>
+          <Route path="/register">
+            <RegisterForm />
+          </Route>
+          <Route path="/login">
+            <LoginForm />
+          </Route>
+          <Route path="/dashboard">
+            <Dashboard />
+          </Route>
+          <Route path="/dashboard-chart">
+            <DashboardChart />
+          </Route>
+          <Route path="/about">
+            <About />
+          </Route>
+          <Route path="/">
+            <Container />
+          </Route>
+        </Switch>
+        <Footer />
+      </div>
+      <ReactQueryDevtools />
```

Comment:
notice this doesn't show up in a production environment ... it has been added as a devDependency only